generate-certs

```diff
@@ -66,7 +68,8 @@ cp ${BASE_SSL_CONF} ${conf}
 
 openssl genrsa -out private/koji_ca_cert.key 2048
 
-sed -i "s/email\:move/DNS.1:koji-hub,DNS.2:${KOJI_HUB_IP}/" ${conf}
+sed -i "s/dns1name/koji-hub/" ${conf}
```

Comment:
I think the reason I don't see this error is because I often run rhel based osbs-box. I haven't had a chance to look into this further, but if we add both `ip1name` and `dns2name` both pointing to the IP address, it might solve for fedora and rhel.